### PR TITLE
Rename Html:add() to Html:addHtml()

### PR DESCRIPTION
**Pull request by JanTvrdik**
https://twitter.com/jakubvrana/status/733655999618404352

### CODE REVIEW

File: src/Utils/Html.php [hunk 1]
```diff
@@ -363,17 +363,39 @@ public function getText()
...
+	}
+
+
+	/**
 	 * Adds new element's child.
 	 * @param  Html|string Html node or raw HTML string
 	 * @return self
 	 */
-	public function add($child)
+	public function addHtml($child)
```
Comment (enumag) [inline src/Utils/Html.php]: Maybe only allow Html instance?

Comment (fprochazka) [inline src/Utils/Html.php]: Not sure that's a good idea. How would you provide custom HTML as string?

Comment (JanTvrdik) [inline src/Utils/Html.php]: You would need to wrap it as an Html instance. But I think that the current solution is better.

**Comment by JanTvrdik** [comment]
Other places which may lead to confusion:

* `$el->insert` – maybe not used by anyone directly
* `$el[]` – likely used by somebody, don't know how to solve except for deprecating the entire `ArrayAccess` implementation

File: src/Utils/Html.php [hunk 2]
```diff
@@ -363,17 +363,39 @@ public function getText()
 
 
 	/**
+	 * @deprecated
+	 */
+	public function add($child)
```
Comment (milo) [inline src/Utils/Html.php]: I like how is the add() short. What about to allow HTML instance and strings consider as a text? Not sure how to warn about change.

**Comment by castamir** [comment]
@JanTvrdik 
`$el->insert()` might became protected or should at least be annotated as `@internal`
`$el[]` is a shortcut for adding child elements, isn't it? We can modify array access setter to use `addHtml()`... or we can deprecate this feature if noone actually use it :D

**Pull request merged**